app/src/test/java/org/mozilla/tv/firefox/architecture/KillswitchLinearLayoutTest.kt

```diff
@@ -0,0 +1,124 @@
+package org.mozilla.tv.firefox.architecture
+
+import android.app.Activity
+import android.view.View
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mozilla.tv.firefox.components.locale.LocaleManager
+import org.robolectric.Robolectric
+import org.robolectric.RobolectricTestRunner
+import java.util.Locale
+
+@RunWith(RobolectricTestRunner::class)
+class KillswitchLinearLayoutTest {
+
+    private lateinit var killswitchLayout: KillswitchLinearLayout
+
+    @Before
+    fun setup() {
+        val activityController = Robolectric.buildActivity(Activity::class.java)
+        killswitchLayout = KillswitchLinearLayout(activityController.get())
+    }
+
+    @Test
+    fun `GIVEN requirements are not set WHEN visibility is set THEN visibility should always be gone`() {
+        // Don't set requirements
+        visibilityIsAlwaysGone()
```

Comment:
nit: `assertVisibilityIsAlwaysGone`. Without following the naming convention, I have to double-check the definition to ensure it's doing what I expect.